.eslintrc.js

```diff
@@ -0,0 +1,267 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es6": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 2018
+    },
+    "rules": {
```

Comment:
Can you explain where does this huge set of rules come from ?

Reply:
The rules were auto-generated by eslint. However, it looks like we only need three exceptions from the "recommend" defaults, I shall revise this.